### ISSUE TITLE
Package on pypi not updated

### ISSUE THREAD

**Issue by lgbaldoni** [Toilal/rebulk#28]
Still at [3.2.0](https://pypi.org/project/rebulk/): is there a reason?

**Comment by Toilal** [comment]
Yes, I have an issue with github workflow and semantic release, I'll try to fix this in the next few days.

**Comment by Toilal** [comment]
Resolved — releases are now cut automatically via python-semantic-release with PyPI trusted publishing (OIDC). 4.2.0 and 4.2.1 are live on PyPI: https://pypi.org/project/rebulk/
